README.md

```diff
@@ -38,6 +38,32 @@ OpenColorIO is free and open source software ([LICENSE](LICENSE)), and
 one of several projects actvively sponsored by the ASWF
 ([Academy Software Foundation](https://www.aswf.io/)).
 
+OpenColorIO Project Mission
+---------------------------
+
+The OpenColorIO project is committed to providing an industry standard solution 
+for highly precise, performant, and consistent color mangement across digital 
+content creation applications and pipelines.
+
+OpenColorIO aims to:
+
+* be stable, secure, and thouroughly tested on all platforms
+* be performant on modern CPUs and GPUs
+* be simple, scalable, and well documented
+* be compatible with critical color and imaging standards
+* provide lossless color processing wherever possible
+* maintain config backwards compatability across major versions
+* have every new feature carefully reviewed by industry leaders
```

Comment:
Should we clarify what we mean by "industry"?  Should we assert a film/tv/games focus?

Reply:
Yes, agreed.

Reply:
`mangement` —> management

Reply:
Changed and fixed.